models/src/Ballom.cpp

```diff
@@ -34,10 +34,11 @@ Ballom::~Ballom(){
 
 void Ballom::changeMoveState()
 {
-	this->moveState =(EnemyMoveState)(rand() % (NUM_MOVEMENT_STATES));
+	int num = RNG::getRandomNumber(0,NUM_MOVEMENT_STATES-1);
+	this->moveState =(EnemyMoveState)(num);
```

Comment:
I guess this was added by someone else, but why are there parentheses around EnemyMoveState?